frontend/src/scenes/experiments/ExperimentView/SummaryTable.tsx

```diff
@@ -88,13 +88,68 @@ export function SummaryTable(): JSX.Element {
             title: 'Conversion rate',
             render: function Key(_, item): JSX.Element {
                 const conversionRate = conversionRateForVariant(experimentResults, item.key)
-                return (
-                    <div className="font-semibold">
-                        {conversionRate === '--' ? conversionRate : `${conversionRate}%`}
-                    </div>
-                )
+                if (!conversionRate) {
+                    return <>--</>
+                }
+
+                return <div className="font-semibold">{`${conversionRate.toFixed(2)}%`}</div>
             },
-        })
+        }),
+            columns.push({
+                key: 'delta',
+                title: (
+                    <div className="inline-flex items-center space-x-1">
+                        <div className="">Delta %</div>
+                        <Tooltip title="Delta % indicates the percentage change in the conversion rate between the control and the test variant.">
+                            <IconInfo className="text-muted-alt text-base" />
+                        </Tooltip>
+                    </div>
+                ),
+                render: function Key(_, item): JSX.Element {
+                    if (item.key === 'control') {
+                        return <></>
+                    }
+
+                    const controlConversionRate = conversionRateForVariant(experimentResults, 'control')
+                    const variantConversionRate = conversionRateForVariant(experimentResults, item.key)
+
+                    if (!controlConversionRate || !variantConversionRate) {
+                        return <>--</>
+                    }
+
+                    const delta = variantConversionRate - controlConversionRate
+
+                    return (
+                        <div
+                            className={`font-semibold ${delta > 0 ? 'text-success' : delta < 0 ? 'text-danger' : ''}`}
+                        >{`${delta > 0 ? '+' : ''}${delta.toFixed(2)}%`}</div>
+                    )
+                },
+            }),
+            columns.push({
+                key: 'credibleInterval',
+                title: (
+                    <div className="inline-flex items-center space-x-1">
+                        <div className="">Credible interval</div>
+                        <Tooltip title="A credible interval represents the range within which the true parameter value lies with 95% probability, given the observed data.">
```

Comment:
```suggestion
                        <Tooltip title="A credible interval represents a range within which we believe the true parameter value lies with a certain probability (often 95%), based on the posterior distribution derived from the observed data and our prior beliefs.">
```

I'm being pedantic here, but maybe consider being a touch more explicit in this tooltip.  My proposal incorporates the following elements:

- emphasizes that the interval is based on the posterior distribution
- mentions that prior beliefs are also incorporated
- avoids implying that 95% is the only possible probability level

Feel free to ignore, but just figured I'd bring it up.

Reply:
nice, thanks! I think we still need to explain prior/posterior in the docs (issue: https://github.com/PostHog/posthog/issues/24277), but I'll use this for now :)